charts/netbird/templates/management-roles.yaml

```diff
@@ -2,23 +2,25 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   name: {{ include "netbird.fullname" . }}-management
+  namespace: {{ .Release.Namespace | quote }}
   labels:
     {{- include "netbird.management.labels" . | nindent 4 }}
 rules:
 - apiGroups: [""]
   resources: ["secrets"]
-  resourceNames: [{{ include "netbird.fullname" . }}-management]
```

Comment:
This should not be removed

Reply:
by setting this, are you not assuming that all of your secrets are defined in a secret name called {{ include "netbird.fullname" . }}-management?

Reply:
Yes, I am using this to avoid that the service account could have access to other secrets. Least privilege principle.

Reply:
Maybe we can get rid of the management suffix, as it also includes relay secrets..

Reply:
i am using multiple secrets for different values as they are produced by different upstream creators.  how about we use a values file value and default it to {{ include "netbird.fullname" . }}-management, then i can override with *?

Reply:
see last commit, to see if it makes sense

Reply:
I am thinking in the scenario where netbird is deployed in a shared namespace. For security reasons I prefer to stick the secret name that matches the release name,

Reply:
Ok, makes sense to be able to override the secret name.